infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogEntryWriter.java

```diff
@@ -74,12 +75,13 @@ private boolean applyMsg(LogReplicationEntryMsg txMessage) {
         // value cannot be used here as its value needs to be changed in the lambda function below.
         AtomicBoolean registryTableUpdated = new AtomicBoolean(false);
         List<OpaqueEntry> opaqueEntryList = CorfuProtocolLogReplication.extractOpaqueEntries(txMessage);
-
+        log.warn("Number of opaqueEntries in the logEntry batch is {}", opaqueEntryList.size());
```

Comment:
I don't think this should be a warning, it can be changed to debug. Also we can probably simplify the message to make it shorter. Something like: Total opaqueEntries count={}